python/manylinux1/build_arrow.sh

```diff
@@ -111,6 +111,7 @@ PATH="$PATH:${CPYTHON_PATH}/bin" $PYTHON_INTERPRETER setup.py build_ext \
     --bundle-boost \
     --boost-namespace=arrow_boost
 PATH="$PATH:${CPYTHON_PATH}/bin" $PYTHON_INTERPRETER setup.py bdist_wheel
+# Source distribution is used for debian pyarrow packages.
```

Comment:
Is there any documentation about this?

Reply:
pyarrow Debian package doesn't exist.
Source distribution is needed to use pyarrow with libarrow Debian package.

Reply:
Could you care to explain, I'm missing some pieces. My comment was regarding uwe comment on zulip.

Reply:
In any case, I'm not sure why this needs to be there. Anyone can build a source distribution from scratch (as the name suggests, it just packages source code together).

Reply:
This gives us a reproducible and controlled environment to build a source tarball. You still need the right versions of setuptools amd friends to actually support Markdown in the description field of the package information.